Documentation/Books/Manual/Upgrading/Manually/Cluster.md

```diff
@@ -0,0 +1,129 @@
+--> TMP <---
+
+
+Upgrading a Cluster
+===================
+
+This page will guide you through the process of an in-place upgrade (see also
+[General Upgrade Information](../GeneralInfo/README.md)). The different
+nodes in a cluster can be updated one at a time without incurring downtime
+of the cluster and very short downtimes of the single nodes.
+
+
+Upgrading a Cluster on Debian & Ubuntu
+======================================
+
+Please note that this procedure is not guaranteed to work for upgrades between minor versions, e.g. from `3.2.x` to `3.3.x`, but only for upgrades between patch versions, e.g. from `3.2.6` to `3.2.12`.
+
+Preparations
+------------
+
+The ArangoDB installation packages (e.g. for Debian or Ubuntu) set up a
+convenient standalone instance of `arangod`. During installation, this instance's
+database will be upgraded (see [`--database.auto-upgrade` in General
+Options](../../Administration/Configuration/GeneralArangod.md#database-upgrade))
+and the service will be (re)started.
+
+You have to make sure that your cluster deployment is independent of this
+standalone instance. Specifically, make sure that the database directory as
+well as the socket used by the standalone instance provided by the package are
+separate from the ones in your cluster configuration. Also, that you haven't
+modified the init script or systemd unit file for the standalone instance in way
+that it would start or stop your cluster instance instead.
+
+You can read about the details on how to deploy your cluster indendently of the
+standalone instance in the [cluster deployment preliminary](../../Deployment/Cluster/PreliminaryInformation.md).
+
+In the following, we assume that you don't use the standalone instance from the
+package but only a cluster instance, and we will move the standalone instance
+out of the way if necessary so you have to make as little changes as possible to
+the running cluster.
+
+Install the new version
+-----------------------
+
+Now you can install the new package version. You don't have to stop the cluster
+process(es) before doing that.
+
+For example, if you want to upgrade to `3.2.12-1` on Debian or Ubuntu, either call
+
+```
+$ apt install arangodb=3.2.12-1
+```
+
+(`apt-get` on older versions) if you've added the ArangoDB repository. Or
+install a specific package using
+
+```
+$ dpkg -i arangodb3-3.2.12-1_amd64.deb
+```
+
+after you've downloaded the file. See https://download.arangodb.com/.
```

Comment:
Is this link intentionally pointing to the download archive rather then the normal homepage?
https://www.arangodb.com/download/ (marketing would probably prefer the latter)